src/mmw/js/src/core/modals/templates/:w

```diff
@@ -0,0 +1,26 @@
+<div class="modal-dialog modal-dialog-sm">
```

Comment:
Looks like a copy of the share modal may have been accidentally committed as a new template named `:w`.

Reply:
Oops, deleted this and forgot to push the change.